ruff_cli/src/main.rs

```diff
@@ -159,8 +160,16 @@ we'd be very appreciative!
     };
 
     if let Some(code) = cli.explain {
-        commands::explain(&code, format)?;
-        return Ok(ExitCode::SUCCESS);
+        match Rule::from_code(&code) {
+            Ok(rule) => {
+                commands::explain(rule, format)?;
+                return Ok(ExitCode::SUCCESS);
+            }
+            Err(error) => {
+                eprintln!("{error}");
+                return Ok(ExitCode::FAILURE);
```

Comment:
Is this intended? It should all be handled by the outer function via `commands::explain(&code, format)?;`.

Reply:
With the previously described change to use `value_parser` this diff has been removed.